extension-manifest-v3/src/background/utils/map.js

```diff
@@ -69,7 +69,11 @@ export default class AutoSyncingMap {
     // and log it. A potential improvement could be to treat the
     // in-memory map as the source of truth in that scenario.)
     this._pending = new Promise((resolve, reject) => {
-      chrome.storage.local.get([this.storageKey], (result) => {
+      // Migration to session storage
+      // TODO: Remove this code after a few releases
+      chrome.storage.local.remove(this.storageKey);
+
+      chrome.storage.session.get([this.storageKey], (result) => {
```

Comment:
After switching to session storage, please update the comment on the class ("with local storage from time to time" -> "with session storage from time to time")